app.py

```diff
@@ -3,60 +3,73 @@
 from flask import Flask, render_template, request, redirect
 import dateutil.parser
 
-app = Flask(__name__, static_url_path='/static')
-app.config.from_object('flask_config.Config')
+from viewmodel import ViewModel
 
+def createListWithStatus(status, itemList):
+    return [TodoItem(x['name'], status, x['id'], duedate=x['due'], last_modified=x['dateLastActivity']) for x in
+            itemList]
 
-# Jinja filters
-@app.template_filter()
-def format_datetime(value):
-    fromisoformat = dateutil.parser.parse(value)
-    return fromisoformat.strftime("%b %d")
 
+def create_app():
+    app = Flask(__name__, static_url_path='/static')
+    app.config.from_object('flask_config.Config')
```

Comment:
Unrelated to this PR's changes, but your app doesn't actually use TODO_BOARD_ID - you set it but never use it for any requests. So any reference to it should be removed.

You should also remove TODO_LIST_ID etc from flask_config.py as they're not used from there. You use os.getenv again elsewhere

Reply:
Refactored and removed the LIST ID completely and now it uses the TODO_BOARD_ID